internal/api/handlers.go

```diff
@@ -404,3 +450,18 @@ var GeneratePassword = func(length int) (string, error) {
 	}
 	return string(b), nil
 }
+
+// Helper function to generate a JWT
+var GenerateJWT = func(username, jwtSecret string, permissions int) (string, error) {
```

Comment:
lets not do function declarations like this unless its anonymous or higher order
```suggestion
function GenerateJWT(username, jwtSecret string, permissions int) (string, error) {
```

Reply:
Done